src/app/layout.tsx

```diff
@@ -1,15 +1,22 @@
 import type { PropsWithChildren } from 'react'
+import { Noto_Sans_KR } from 'next/font/google'
 
 import QueryProvider from '@/hooks/QueryProvider'
 
 import './globals.css'
 
+const notoSans = Noto_Sans_KR({ subsets: ['latin'], weight: ['400', '500', '700'] })
+
```

Comment:
```suggestion
const notoSans = Noto_Sans_KR({
  subsets: ['latin'],
  variable: '--font-notoSansKR',
})
```

가변 글꼴이라, 따로 fontWeight 지정 안해줘도 됩니다! 
tailwindCss 와 함께 사용할때는 classname 대신 variables 로 공식문서에서는 지정하네요!

https://nextjs.org/docs/app/building-your-application/optimizing/fonts#with-tailwind-css

Reply:
네. 해당 사항 반영해서 다시 올렸습니다.

Reply:
폰트 굵기를 ["400", "500", "700"]으로 고정하고싶었는데, 해당 방식이 아닌 tailwind에서 지정하는 것이군요.

수정해서 올렸습니다.